code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -186,13 +186,15 @@
 	..()
 
 /mob/living/carbon/human/proc/Stasis(amount)
-	if((species.flags & NO_SCAN) || isSynthetic())
+	//if((species.flags & NO_SCAN) || isSynthetic()) vorestation removal
+	if(((species.flags & NO_SCAN) && !istype(species, /datum/species/xenochimera)) || isSynthetic()) //vorestation addition

```

Comment:
Actually I should point out, in_stasis may not do what you want it to do anymore, you may want to re-evaluate your use of it in general if you just want to "stop life ticks" while they regenerate. I notice there's a 'transforming' var that causes life ticks to just return at the top. You might want to consider using that, rather than wasting time dividing some number by ten thousand every tick.